### PR TITLE
chore(ci): cspell improve error readability

### LINKED ISSUE #288: 🛠️ [TASK] : Spell checking error logs readability
### Summary

When earthly +check-spelling fails its hard to find what went wrong 

### Description

Here is example of error outcome of `earthly +check-spelling`
```bash
  ================================== ❌ FAILURE ===================================
       +check-spelling *failed* | 354/843 ./catalyst_voices/packages/catalyst_voices_brands/lib/src/theme_extensions/brand_assets.dart 15.39ms
     +check-spelling *failed* | 355/843 ./catalyst_voices/packages/catalyst_voices_brands/lib/src/theme_extensions/theme_extensions.dart 6.15ms
     +check-spelling *failed* | 356/843 ./catalyst_voices/packages/catalyst_voices_brands/lib/src/theme_extensions/voices_color_scheme.dart 31.48ms
     +check-spelling *failed* | 357/843 ./catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/catalyst.dart 67.90ms X
     +check-spelling *failed* | ./catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/catalyst.dart:394:8 - Unknown word (endregion)
     +check-spelling *failed* | 358/843 ./catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/fallback.dart 12.60ms
     +check-spelling *failed* | 359/843 ./catalyst_voices/packages/catalyst_voices_brands/lib/src/utils/typedefs.dart 6.44ms
     +check-spelling *failed* | 360/843 ./catalyst_voices/packages/catalyst_voices_brands/pubspec.yaml 8.05ms
     +check-spelling *failed* | 361/843 ./catalyst_voices/packages/catalyst_voices_brands/README.md 3.81ms
     +check-spelling *failed* | CSpell: Files checked: 843, Issues found: 1 in 1 files
     +check-spelling *failed* | earthly debugger | Command /bin/sh -c 'cspell-cli lint . --dot' failed with exit code 1
     +check-spelling *failed* | Entering interactive debugger
     +check-spelling *failed* | ERROR /var/folders/yn/dqkwzjr93zq23nv4yjfd77c00000gn/T/earthly-git3833692372/Earthfile:19:4
     +check-spelling *failed* |       The command
     +check-spelling *failed* |           RUN cspell-cli lint . --dot
     +check-spelling *failed* |       did not complete successfully. Exit code 1
```
btw. this is shorter version, full is available [here](https://github.com/input-output-hk/catalyst-voices/actions/runs/10249367117/job/28352611522?pr=662).

What you need to find is "unknown" in order to figure out what went wrong.

### PR DESCRIPTION
**Pull request by oldGreg5**
# Description

Cspell needs to be easy readable to find issues fast.

## Related Issue(s)

Fixes #288

## Description of Changes

Minimize spell check output to errors only and fail-fast if one is picked up.
CleanShot 2024-09-24 at 14 07 01

## Please confirm the following checks

* [ ] My code follows the style guidelines of this project
* [ ] I have performed a self-review of my code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have made corresponding changes to the documentation
* [ ] My changes generate no new warnings
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] New and existing unit tests pass locally with my changes
* [ ] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: earthly/cspell/Earthfile [hunk 1]
```diff
@@ -16,8 +16,8 @@ CHECK:
     WORKDIR /work
 
     COPY $src .
-    
-    RUN cspell-cli lint . --dot
+
+    RUN cspell-cli --quiet --fail-fast lint . --dot
```
Comment (kukkok3) [inline earthly/cspell/Earthfile:20]: I would remove `--fail-fast` so we can discover all the linting errors in the first run. What do you think?

Comment (Mr-Leshiy) [reply earthly/cspell/Earthfile:20]: Yea we definitely dont want a `--fail-fast` and collect all issues at ones.

Comment (oldGreg5) [reply earthly/cspell/Earthfile]: Makes sense, done

**Review by kukkok3 — APPROVED** [review]
LGTM

**Pull request merged**